mlflow/store/cli.py

```diff
@@ -103,3 +103,14 @@ def download_artifacts(run_id, artifact_path):
     artifact_repo = get_artifact_repository(artifact_uri)
     artifact_location = artifact_repo.download_artifacts(artifact_path)
     print(artifact_location)
+
+
+@commands.command("download-from-uri")
+@click.option("--artifact-uri", "-a", required=True,
+              help="URI pointing to the artifact file or artifacts directory.")
+def download_artifacts_from_uri(artifact_uri):
```

Comment:
i think this is okay but since we might change the main user function for downloading from URIs (to possibly `data.download_uri`, what do you think about making it experimental? (i'll do it in the marking-experimental PR)

Reply:
I am ok marking this experimental. What do I do to mark this as such? :)